spec.html

```diff
@@ -34170,7 +34170,8 @@ <h1>String.prototype.padEnd ( _maxLength_ [ , _fillString_ ] )</h1>
         <p>This method performs the following steps when called:</p>
         <emu-alg>
           1. Let _O_ be ? RequireObjectCoercible(*this* value).
-          1. Return ? StringPad(_O_, _maxLength_, _fillString_, ~end~).
+          1. Let _S_ be ? ToString(_O_).
+          1. Return StringPad(_S_, _maxLength_, _fillString_, ~end~).
```

Comment:
Here, you need to

```suggestion
          1. Let _S_ be ? ToString(_O_).
          1. Let _intMaxLength_ be ℝ(? ToLength(_maxLength_)).
          1. If _fillString_ is not *undefined*, set _fillString_ to ? ToString(_fillString_).
          1. Return StringPad(_S_, _intMaxLength_, _fillString_, ~end~).
```

because `StringPad` is expecting the second parameter to be an integer and the third parameter to be a String or undefined, and built-in parameters may be any ECMAScript language values.